packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -129,13 +134,16 @@ const getQueryBasedOnLogin = (
   return null;
 };
 
+type ModalCommand = Record<OnboardingVersion, (value: boolean) => unknown>;
```

Comment:
This technically has nothing to do with the MainFeedLayout, can we move it away to another file?